django/contrib/admin/helpers.py

```diff
@@ -214,6 +214,8 @@ def contents(self):
             else:
                 if isinstance(f.remote_field, ManyToManyRel) and value is not None:
                     result_repr = ", ".join(map(six.text_type, value.all()))
+                elif isinstance(f.remote_field, OneToOneField) and value is not None:
```

Comment:
Sorry, what I meant was that the test for the `fk=None` case is useful but `value is not None` seems unneeded.